frontend/src/app/services/patient/patient.service.ts

```diff
@@ -35,10 +35,12 @@ export class PatientService {
       }
     };
 
+    sessionStorage.setItem("patientData", JSON.stringify(patientInfo));
```

Comment:
Vytvořila bych zvlášť funkce pro uložení, získání a smazání `patientData` ze `sessionStorage`, které budou používat stejný klíč, pod kterým si to budou ukládat.